app/(auth)/settings.tsx

```diff
@@ -30,6 +35,11 @@ export default function Settings() {
         label: 'Change my password',
         link: '/password-reset?origin=settings'
       },
+      {
+        icon: 'two-factor-authentication' as const,
+        label: 'Set up two factor auth',
+        link: `${savedInstance}/mfaSetup`
```

Comment:
Unfortunately, this link will not work with the main instance "app.wafrn.net". Right now, the app catches all link navigations to there and tries to open them "in-app" so we would have to replicate the MFA setup UI here somehow, or leave this link as is for the moment.

Reply:
Ah. I tried with the expo app, likely it didn't register the urls hence it was working for me. I'll try to get the setup working as well in the app in the future, just wanted to cover the basics right now

Reply:
Yeah, no problem, I can take on building that UI here